### PR TITLE
Consent MUST be per-browsing profile

### PR DESCRIPTION
**Pull request by ddorwin**
Also cleans up text related to consent and network attacks.

### CODE REVIEW

**Comment by ddorwin** [comment]
This addresses some things I found while working on other PRs.

**Comment by ddorwin** [comment]
This is also consistent with issue #223 and PR #287.

File: encrypted-media-respec.html [hunk 1]
```diff
@@ -3611,6 +3611,12 @@
                 <li><p>Data, including <a def-id="distinctive-identifiers"></a> and <a def-id="distinctive-permanent-identifiers"></a>, transmitted between the client and server can be viewed and/or modified by other entities. See <a href="#user-tracking">User Tracking</a>.</p></li>
               </ul>
             </li>
+            <li>
```
Comment (jdsmith3000) [inline encrypted-media-respec.html]: I don't fully understand the intent of this addition.  It doesn't seem related to browsing profiles, but perhaps does provide an additional threat that warrants secured origins.  Is that the purpose?

Comment (ddorwin) [inline encrypted-media-respec.html]: I moved this from line 3877, which I replaced with a more general reference to this section.

Comment (jdsmith3000) [inline encrypted-media-respec.html]: I see the original comment, and agree this section is more appropriate.  LGTM.
